utils/events.js

```diff
@@ -2,119 +2,100 @@ const firestore = require('@google-cloud/firestore');
 const {BigQuery} = require('@google-cloud/bigquery');
 const {Storage} = require('@google-cloud/storage');
 
-const collectionNameList = ['participants','biospecimen', 'boxes', 'module1_v1', 'module1_v2', 'module2_v1', 'module2_v2', 'module3_v1', 'module4_v1', 'bioSurvey_v1', 'menstrualSurvey_v1', 'clinicalBioSurvey_v1','covid19Survey_v1'];
-
-const importToBigQuery = async (event, context) => {
-    const gcsEvent = event;
-    let tableName = ''
-    if (gcsEvent.name.indexOf('.export_metadata') === -1) return true;
-    for (const collectionName  of collectionNameList) {
-        if (gcsEvent.name.includes(collectionName)) {
-            tableName = collectionName;
-            break;
-        }
-    }
+const collectionNameArray = ['participants','biospecimen', 'boxes', 'module1_v1', 'module1_v2', 'module2_v1', 'module2_v2', 'module3_v1', 'module4_v1', 'bioSurvey_v1', 'menstrualSurvey_v1', 'clinicalBioSurvey_v1','covid19Survey_v1', 'kitAssembly'];
```

Comment:
For issue https://github.com/episphere/connect/issues/766 i added `sendgridTracking` collection. Should we add it  into  `collectionNameArray`

Reply:
I don't decide whether it should be synced to BQ, and I didn't receive such request.

Reply:
Got it. Should update later